ractor_cluster/src/node/node_session/mod.rs

```diff
@@ -947,6 +957,9 @@ impl Actor for NodeSession {
             Self::Msg::GetAuthenticationState(reply) => {
                 let _ = reply.send(state.auth.is_ok());
             }
+            Self::Msg::GetInitialSyncState(reply) => {
```

Comment:
We should also add a new event handler method to the `NodeEventSubscription` type, along with a new `NodeServerMessage` which supports the "init done" notification you're adding here.

For backwards compat, you can make the `NodeEventSubscription` trait have a default, empty, implementation.